### PR TITLE
Prime factors altered to use templates

### PR DESCRIPTION
**Pull request by Sloff**
Fixes part of #1163 

Before:
before1
before2
before3

After:
after1
after2
after3

Bug:
Is cached is ```Math::BigInt``` instead of ```0```, not sure why or if it matters
problem

### CODE REVIEW

**Comment by javathunderman** [comment]
The "is unsafe= 0," is alright, I see that a lot. @MrChrisW do you see anything wrong with this?

**Comment by mintsoft** [comment]
@Sloff The `is_cached` is fine, it's just indicating that it's a Math::BigInt rather than a native int.

**Comment by mintsoft** [comment]
I thought the `is unsafe` is for USFW content not html unsafe @moollaza ?

**Comment by moollaza** [comment]
You're correct. It's a flag for IAs that are known to return adult content. Those IAs can only trigger when the user disables safe search.

File: lib/DDG/Goodie/PrimeFactors.pm [hunk 1]
```diff
@@ -52,10 +60,10 @@ sub format_prime {
     my @html_result = ();
```
Comment (moollaza) [inline lib/DDG/Goodie/PrimeFactors.pm]: I believe we can remove this now (and the related html code)

File: lib/DDG/Goodie/PrimeFactors.pm [hunk 2]
```diff
@@ -71,6 +79,24 @@ sub commify {
...
+sub format_answer {
+    my ($plaintext, $title, $subtitle) = @_;
+    
+    return $plaintext,
+    structured_answer => {
+        id => 'prime_factors',
+        name => 'Answer',
+        data => {
+            title => $title || $plaintext,
+            subtitle => $subtitle
```
Comment (moollaza) [inline lib/DDG/Goodie/PrimeFactors.pm]: Now that we're not dealing with proper sentences, lets' ditch the leading "The" on the subtitle.

**Comment by moollaza** [comment]
@Sloff this is looking great.

@abeyang any thoughts? I suggested we drop "The " from the `subtitle` now that we're not giving an answer in sentence form.

**Comment by Sloff** [comment]
Thanks @moollaza, PS changes made as suggested

**Comment by abeyang** [comment]
@Sloff , looking good! I know we're not consistent with this yet, but I'd like our IAs to have this kind of format to the question, "What is the *type* of *specifics*?"

> **Answer**
> *Specifics* - *Type*

Thus, to the question, "What is the prime factor of 30?"

> **2 x 3 x 5**
> 30 - Prime Factor

**Comment by Sloff** [comment]
Hi @abeyang, changes made as suggested:
after4

**Comment by abeyang** [comment]
:+1:

**Comment by mintsoft** [comment]
Thanks again @Sloff !

**Comment by Sloff** [comment]
Pleasure!

**Comment by moollaza** [comment]
@Sloff Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Pull request merged**